dcos/packagemanager.py

```diff
@@ -428,7 +428,7 @@ def marathon_json(self, options):
         """
 
         params = {"packageName": self.name()}
-        params["packageVersion"] = self.package_version()
+        params["packageVersion"] = self.version()
```

Comment:
It seems weird to me to have the previous line set `packageName` inside the dictionaty definition, and then set this one outside of it. Can we change this to:
```
-        params = {"packageName": self.name()}
-        params["packageVersion"] = self.package_version()
+        params = {
+            "packageName": self.name(),
+            "packageVersion": self.version()
+        }
```